tests/DetectorTest.php

```diff
@@ -302,6 +302,23 @@ public function testAllowArrayMappingWithArrayExtension(): void
         );
     }
 
+    public function testCheckForMagicArrayConstants(): void
+    {
+        $option = $this->createOption();
+        $option->setExtensions([new ArrayExtension()]);
+        $detector = $this->createDetector($option);
+
+        $fileReport = $detector->detect(FileReportTest::getTestFile('test_3'));
+
+        $this->assertContains(
```

Comment:
It looks like the actual result is `$fileReport->getEntries()` and get entire arrays.

I think it should use the `assertSame` assertion to check whether the expected array is same as actual.